common/tags.go

```diff
@@ -0,0 +1,168 @@
+package common
```

Comment:
This appears to be isolated to provide tagging functionality, so rather than `common`, I'm curious if we should call this  package `tagging`? The APIs therein would be more descriptive too (eg. `tagging.FromSSM()`). Thoughts?

Reply:
Sounds good! This is such a minor feature that I didn't know if a separate package for it would be warranted.